### ISSUE TITLE
[RFC]: Array Matrix Math (Possible? Documentation?)

### ISSUE THREAD

**Issue by danieldjewell** [stdlib-js/stdlib#445]
### Description

One of the nice (if not obligatory) features of Numpy is the ability to perform basic arithmetic on ndarrays ... (e.g. ``` array1+array2 ```) [Granted, the reason this is relatively easy to implement in Python is because allows you to provide methods to handle the use of operators... I don't think JavaScript has the same option?] 

Is there a way to do this with stdlib arrays (of any kind)? Right now, doing an ```array1+array2``` results in ```NaN```. I searched the documentation and could not find a thing on this. 

At a minimum, I think a "quick start" or a cheatsheet type document would be *extraordinarily* helpful. Especially a "stdlib math for Numpy users" type document demonstrating some basics... something like:
* ```array = np.array([1,2,3,4])``` == ```var array = array([1,2,3,4]);```
* ```array.max()``` == ```???```
* ```array1+array2``` == ```???```
* etc.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

**Comment by kgryte** [comment]
@danieldjewell Agreed! A NumPy-to-stdlib cheatsheet is something we'd like to add. I've added it to our internal tracker.

Regarding element-wise addition, in your example, `array1 + array2`, unfortunately, this is not possible atm in JavaScript due to the lack of operator overloading. We are in the process, however, of adding functional APIs (e.g., `add(x, y)`) similar to Julia's `+(x,y)` ability.

Are there particular operations that you need now above others?
